src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -111,44 +113,37 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
               {materialType}
             </div>
           </div>
+          {statusBadgeComponentMobile || ""}
```

Comment:
Not a biggie but I think it is more standard to emit `null`if you want nothing rendered in React

Reply:
Great comment, thx!